scripts/findUnusedHelpers.ts

```diff
@@ -0,0 +1,74 @@
+import { join } from "node:path"
+import { fromPackageRoot, readPackageJson } from "@re-/node"
+import { Node, Project, ts } from "ts-morph"
+import { PackageJson } from "type-fest"
+import { getEntryPointsToRelativeDtsPaths } from "./docgen/utils.js"
+
+export type ApiExports = {
+    path: string
+    exportedDeclarations: string[]
+}
+
+export const getPublicApiExports = (project: Project): ApiExports[] => {
+    const rootDir = fromPackageRoot("@re-")
+    const publicApis = ["assert", "model"]
+    const exportAllRegex = /export \*/
+    const apiExports: ApiExports[] = []
+    publicApis.forEach((knownPublicApi) => {
```

Comment:
Let's use a `for...of` loop here to match the style of the rest of the repo. We can also rename `knownPublicApi` here 🤣